Usiqee/Service/ServiceArtist.swift

```diff
@@ -37,4 +38,43 @@ class ServiceArtist {
             }
         }
     }
+    
+    static func follow(artist: Artist, completion: @escaping (Error?) -> Void)  {
+        guard let user = Auth.auth().currentUser else {
+            return
+        }
+        let data: [String : Any] = [
+            "artistId": artist.id,
+            "name": artist.name,
+            "avatar": artist.avatar
+        ]
```

Comment:
Can add direclty this part in Artist Model as relatedData ?

Reply:
indeed 👍